src/components/screens/consul/Debates/DebateDetail.js

```diff
@@ -150,25 +150,26 @@ export const DebateDetail = ({ data, onRefresh, route, navigation }) => {
           />
         )}
 
-        <Wrapper>
-          <Input
-            multiline
-            minHeight={50}
-            name="comment"
-            label={texts.consul.commentLabel}
-            placeholder={texts.consul.comment}
-            autoCapitalize="none"
-            control={control}
-          />
-          <WrapperVertical>
-            <Button
+        <Wrapper style={styles.input}>
+          <WrapperRow>
+            <Input
+              multiline
+              minHeight={50}
+              name="comment"
+              label={texts.consul.commentLabel}
+              placeholder={texts.consul.comment}
+              autoCapitalize="none"
+              control={control}
+              chat
+            />
+            <TouchableOpacity
               onPress={handleSubmit(onSubmit)}
-              title={
-                loading ? texts.consul.submittingCommentButton : texts.consul.commentAnswerButton
-              }
+              style={styles.button}
```

Comment:
what do you think about hiding the keyboard after submission of comments? i felt a bit surprised, that it was still open after sending a test comment.